x-pack/plugins/security_solution/public/management/pages/blocklist/view/blocklist.tsx

```diff
@@ -10,29 +10,7 @@ import { i18n } from '@kbn/i18n';
 import { useHttp } from '../../../../common/lib/kibana';
 import { ArtifactListPage, ArtifactListPageProps } from '../../../components/artifact_list_page';
 import { BlocklistsApiClient } from '../services';
-
-// FIXME:PT delete this when real component is implemented
-const TempDevFormComponent: ArtifactListPageProps['ArtifactFormComponent'] = (props) => {
-  // For Dev. Delete once we implement this component
-  // @ts-ignore
-  if (!window._dev_artifact_form_props) {
-    // @ts-ignore
-    window._dev_artifact_form_props = [];
-    // @ts-ignore
-    window.console.log(window._dev_artifact_form_props);
-  }
-  // @ts-ignore
-  window._dev_artifact_form_props.push(props);
-
-  return (
-    <div>
-      <div style={{ margin: '3em 0' }}>
-        {props.error ? props.error?.body?.message || props.error : ''}
-      </div>
-      {`TODO: ${props.mode} Form here`}
-    </div>
-  );
-};
+import { CreateBlockListForm } from './components/create_blocklist_form';
```

Comment:
Since this form is going to be used for edit/create, should this be renamed to `BlocklistForm` ?